www/js/coolpoints-util.js

```diff
@@ -72,11 +83,14 @@ function animatePointUpdate(ptEl, msgEl, diff) {
   const isPositive = diff > 0;
   const bounceAnimationName = isPositive ? "cpBounce" : "cpShake";
   const fadeAnimationName = isPositive ? "cpFadeGreen" : "cpFadeRed";
+  const glowAnimationName = isPositive ? "glowGreen" : "glowRed";
   const msgText = isPositive ? `+${diff}` : `${diff}`;
-
+  const btnEl = $("#coolpointsbtn");
+  
   msgEl.text(msgText);
-  updateAnimation(ptEl.attr("id"), bounceAnimationName);
-  updateAnimation(msgEl.attr("id"), fadeAnimationName);
+  updateAnimation(btnEl.attr("id"), [glowAnimationName]);
+  updateAnimation(ptEl.attr("id"), [bounceAnimationName]);
+  updateAnimation(msgEl.attr("id"), [fadeAnimationName]);
```

Comment:
question: why were these changed to pass in an array instead of a string?

Reply:
I was testing something involving multiple effects, but forgot to switch this part back after reverting those changes, lol.